variables.tf

```
@@ -80,3 +80,15 @@ variable "add_attribute_tag" {
   default     = true
   description = "If `true` will add additional attribute tag to the requester and accceptor resources"
 }
+
+variable "aws_route_create_timeout" {
+  type        = string
+  default     = "2m"
```

Default is 5m on the resource so let's keep the same default

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route#create

```suggestion
  default     = "5m"
```